packages/browser-wallet/src/popup/popupX/constants/routes.ts

```diff
@@ -219,6 +224,14 @@ const buildAbsoluteRoutes = <R extends RouteNode | RoutePath | RouteChildren>(
 
 export const absoluteRoutes = buildAbsoluteRoutes(relativeRoutes);
 
+export const transactionDetailsRoute = (account: AccountAddress.Type, tx: TransactionHash.Type) =>
+    absoluteRoutes.home.transactionLog.details.path
```

Comment:
**Suggestion (non-blocking):** In case you have not seen, React router have a function for applying arguments [`generatePath`](https://reactrouter.com/en/main/utils/generate-path), although I like the simplicity of just doing this.

Reply:
I did not know this, so thanks for bringing that up 👍 I still think it's nice to have these helpers but will use the `generatePath` in the body.